test/java/seedu/addressbook/parser/UtilsTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.addressbook.parser;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.common.Utils;
+
+import java.util.ArrayList;
+
+public class UtilsTest {
+
+    @Test
+    public void testIsAnyNull() {
+        assertEquals(false, Utils.isAnyNull("TEST"));
+    }
+    
+    @Test
+    public void testElementsAreUnique_returnTrue() {
+        ArrayList<Object> testCases = new ArrayList<Object>();
+        testCases.add("TEST");
```

Comment:
going further you can consider moving all the steps necessary for the test to run (or preconditions) to a set up method before beginning the tests; correspondingly can put all the tear down in a method after the tests.